<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
    </line>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
    </line>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
    </line>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
    </line>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"><div  class="fail"><span class="fail-label">F </span>expected:<9> but was:<7></div></div>]]>
    </line>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"><div  class="fail"><span class="fail-label">F </span>expected:<9> but was:<7></div></div>]]>
    </line>
    <line>
      <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
    </line>
  </reporter-output>
  <suite started-at="2019-04-18T13:11:35Z" name="Default suite" finished-at="2019-04-18T13:11:35Z" duration-ms="221">
    <groups>
      <group name="testEntityKey:QME-TC-7">
        <method signature="CalculatorTest.testDivison()[pri:0, instance:TestNGmavenexemple.CalculatorTest@3c7f66c4]" name="testDivison" class="TestNGmavenexemple.CalculatorTest"/>
      </group> <!-- testEntityKey:QME-TC-7 -->
      <group name="testEntityKey:QME-TC-6">
        <method signature="CalculatorTest.testSum()[pri:0, instance:TestNGmavenexemple.CalculatorTest@3c7f66c4]" name="testSum" class="TestNGmavenexemple.CalculatorTest"/>
      </group> <!-- testEntityKey:QME-TC-6 -->
    </groups>
    <test started-at="2019-04-18T13:11:35Z" name="Default test" finished-at="2019-04-18T13:11:35Z" duration-ms="221">
      <class name="TestNGmavenexemple.CalculatorTest">
        <test-method is-config="true" signature="beforeEachTest()[pri:0, instance:TestNGmavenexemple.CalculatorTest@3c7f66c4]" started-at="2019-04-18T15:11:35Z" name="beforeEachTest" finished-at="2019-04-18T15:11:35Z" duration-ms="15" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeEachTest -->
        <test-method is-config="true" signature="initCalculator()[pri:0, instance:TestNGmavenexemple.CalculatorTest@3c7f66c4]" started-at="2019-04-18T15:11:35Z" name="initCalculator" finished-at="2019-04-18T15:11:35Z" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- initCalculator -->
        <test-method signature="testDivison()[pri:0, instance:TestNGmavenexemple.CalculatorTest@3c7f66c4]" started-at="2019-04-18T15:11:35Z" name="testDivison" finished-at="2019-04-18T15:11:35Z" duration-ms="16" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
            </line>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- testDivison -->
        <test-method signature="testSum()[pri:0, instance:TestNGmavenexemple.CalculatorTest@3c7f66c4]" started-at="2019-04-18T15:11:35Z" name="testSum" finished-at="2019-04-18T15:11:35Z" duration-ms="0" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<9> but was:<7>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<9> but was:<7>
	at org.junit.Assert.fail(Assert.java:93)
	at org.junit.Assert.failNotEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:128)
	at org.junit.Assert.assertEquals(Assert.java:472)
	at org.junit.Assert.assertEquals(Assert.java:456)
	at TestNGmavenexemple.CalculatorTest.testSum(CalculatorTest.java:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"><div  class="fail"><span class="fail-label">F </span>expected:<9> but was:<7></div></div>]]>
            </line>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"><div  class="fail"><span class="fail-label">F </span>expected:<9> but was:<7></div></div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- testSum -->
        <test-method is-config="true" signature="afterEachTest()[pri:0, instance:TestNGmavenexemple.CalculatorTest@3c7f66c4]" started-at="2019-04-18T15:11:35Z" name="afterEachTest" finished-at="2019-04-18T15:11:35Z" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[<div style="display:block;margin-left:10px;"></div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterEachTest -->
      </class> <!-- TestNGmavenexemple.CalculatorTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
